CHANGELOG.md

```diff
@@ -81,6 +81,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (all) [#16497](https://github.com/cosmos/cosmos-sdk/pull/16497) Removed all exported vestiges of `sdk.MustSortJSON` and `sdk.SortJSON`.
```

Comment:
Changelog should be under unreleased.

Reply:
Gotcha, thanks! Let me fix that.

Reply:
Can we have it under an Improvements subcategory?

Reply:
Done, please take a look again.

Reply:
You've put it back where it was.
What I mean is 

```diff
## [Unreleased]

+ ### Improvements
+
+ <changelog>

### API Breaking Changes
```

Reply:
Done, thanks!